config/config.go

```diff
@@ -37,6 +34,7 @@ type Config struct {
 	OTServiceName              string        `envconfig:"OTEL_SERVICE_NAME"`
 	OTBatchTimeout             time.Duration `envconfig:"OTEL_BATCH_TIMEOUT"`
 	OtelEnabled                bool          `envconfig:"OTEL_ENABLED"`
+	LocalstackHost             string        `envconfig:"LOCALSTACK_HOST"`
```

Comment:
Should we really have config specifically for local running? Is this really needed or am I misunderstanding the purpose of this parameter?

Reply:
The localstack container that is used to run this with requires specific authentication parameters to be passed in so using existing methods of connecting to the s3 buckets doesn't work when running locally.  I have followed the pattern that has been established in several other services:
https://github.com/ONSdigital/dp-upload-service/blob/7d502b3bb554fe6a94e39d65a833e490f726193b/service/initialise.go#L84
https://github.com/ONSdigital/dp-static-file-publisher/blob/36bf05ae08b064dcce5f464f40d676486e9a571a/service/initialise.go#L202
https://github.com/ONSdigital/dp-download-service/blob/5d98f0fa6fc45d0e10d1033ab2f619e9871812d2/service/external/external.go#L50